doc/user/shared-content/namespaces-content.md

```diff
@@ -0,0 +1,120 @@
+Namespaces are a way to organize Materialize objects logically. In organizations
+with multiple objects, namespaces help avoid naming conflicts and make it easier
+to manage objects.
+
+## Namespace hierarchy
+
+Materialize follows SQL standard's namespace hierarchy for most objects (for the
+exceptions, see [Other objects](#other-objects)).
+
+|                           |             |
+|---------------------------| ------------|
+| 1st/Highest level:        |  **Database** |
+| 2nd level:                |  **Schema**   |
+| 3rd level:                | <table><tbody><tr><td><ul><li>**Table**</li><li>**View**</li><li>**Materialized view**</li><li>**Connection**</li></ul></td><td><ul><li>**Source**</li><li>**Sink**</li><li>**Index**</li></ul></td><td><ul><li>**Type**</li><li>**Function**</li><li>**Secret**</li></ul></td></tr></tbody></table>|
+| Lowest level:             | **Column**     |
```

Comment:
nit: do we want this to be comparable to the first entry in the table? probably would make it less legible than as currently written.
```suggestion
| 4th/Lowest level:             | **Column**     |
```

Reply:
might also read a little smoother to just not have the number at the front for Highest and Lowest?